src/app/sitemap.ts

```diff
@@ -1,41 +1,149 @@
 import type { MetadataRoute } from "next";
 
 import { client } from "studio/lib/client";
-import { DocumentWithSlug } from "studio/lib/interfaces/global";
-import { PageBuilder } from "studio/lib/interfaces/pages";
-import { DOCUMENTS_WITH_SLUG_QUERY } from "studio/lib/queries/siteMap";
-import { LANDING_PAGE_QUERY } from "studio/lib/queries/siteSettings";
+import {
+  DocumentTranslatedSitemapData,
+  FieldTranslatedSitemapData,
+  SitemapBaseData,
+  UntranslatedSitemapData,
+} from "studio/lib/interfaces/sitemap";
+import { LanguageObject } from "studio/lib/interfaces/supportedLanguages";
+import { LEGAL_DOCUMENTS_SITEMAP_QUERY } from "studio/lib/queries/admin";
+import { PAGES_SITEMAP_QUERY } from "studio/lib/queries/pages";
+import {
+  LANDING_PAGE_SITEMAP_QUERY,
+  LANGUAGES_QUERY,
+} from "studio/lib/queries/siteSettings";
+import {
+  COMPENSATIONS_PAGE_SITEMAP_QUERY,
+  CUSTOMER_CASES_PAGE_SITEMAP_QUERY,
+} from "studio/lib/queries/specialPages";
 import { token } from "studio/lib/token";
```

Comment:
Is it better to add "token as studiotoken" here?

Reply:
Sure :+1: